iam-identity/v1.ts

```diff
@@ -36,7 +36,7 @@ import { getSdkHeaders } from '../lib/common';
  */
 
 class IamIdentityV1 extends BaseService {
-  static DEFAULT_SERVICE_URL: string = 'https://iam.cloud.ibm.com';
+  static DEFAULT_SERVICE_URL: string = 'https://iam.test.cloud.ibm.com';
```

Comment:
I don't think you want to actually make this change in the generated code, do you?  This change was (I'm guessing) because you used an API definition that contained the staging instance's URL as the server url.
```suggestion
  static DEFAULT_SERVICE_URL: string = 'https://iam.cloud.ibm.com';
```